dms-core/src/main/kotlin/team/aliens/dms/domain/manager/usecase/FindManagerAccountIdUseCase.kt

```diff
@@ -15,13 +16,14 @@ class FindManagerAccountIdUseCase(
     private val sendEmailPort: SendEmailPort
 ) {
 
-    fun execute(schoolId: UUID, answer: String): String {
+    fun execute(schoolId: UUID, answer: String): ManagerEmailResponse {
         schoolService.getSchoolById(schoolId)
             .apply { checkAnswer(answer) }
 
         val user = userService.queryUserBySchoolIdAndAuthority(schoolId, Authority.MANAGER)
 
         sendEmailPort.sendAccountId(user.email, user.accountId)
-        return StringUtil.coveredEmail(user.email)
+
+        return ManagerEmailResponse(StringUtil.coveredEmail(user.email))
```

Comment:
p3 여기 파라미터 이름 명시해주시면 좋을 것 같아요
```suggestion
        return ManagerEmailResponse(
             email = StringUtil.coveredEmail(user.email)
         )
```